api/models/templates/app.tmpl

```diff
@@ -350,6 +356,7 @@
     "Value": { "Ref": "AWS::AccountId" }
   },
   "RegistryRepository": {
```

Comment:
Does `RegistryId` also need the condition `RegionHasRegsitry`?  

It seems to me if `RegsitryId` is set, and it appears to me to reference something that will always be there, then this branch will still (always) execute, which we do not wish to happen:  https://github.com/convox/rack/blob/master/api/models/build.go#L223 .